components/eventing-controller/README.md

```diff
@@ -0,0 +1,115 @@
+# Eventing Controller
+
+## Overview
+
+This component contains controllers for various CustomResourceDefinitions related to eventing in Kyma. The controller comes with 2 containers:
+- [controller](https://github.com/kubernetes-sigs/controller-runtime)
```

Comment:
```suggestion
This component contains controllers for various CustomResourceDefinitions related to eventing in Kyma. The controller comes with this container:
- [controller](https://github.com/kubernetes-sigs/controller-runtime)
```

The document says the "controller comes with 2 containers", but only one is mentioned. Is this intentional? I suggest we either add the 2nd one or re-write this.